libvast/src/format/csv.cpp

```diff
@@ -117,16 +117,15 @@ caf::error writer::write(const table_slice& x) {
     append('\n');
     write_buf();
   }
+  auto flat_layout = flatten(layout);
   // Print the cell contents.
   auto iter = std::back_inserter(buf_);
   for (size_t row = 0; row < x.rows(); ++row) {
     append(last_layout_);
-    append(separator);
```

Comment:
Looks like this statement got lost?

Reply:
The output is unchanged. I just simplified the logic from 
```
print(typename)
print(separator)
print(fields[0])
for (size_t i = 1; i < fields.size(); ++i) {
  print(separator)
  print(fields[i])
}
```
to
```
print(typename)
for (size_t i = 0; i < fields.size(); ++i) {
  print(separator)
  print(fields[i])
}
```
We would need to revert that if we added an option to omit the typename.